bot/resources/tags/json.md

```diff
@@ -12,4 +24,4 @@ Looking at how JSON is set up, you can see that they require the following at mi
 }
 ```
 
-If you add at least this to your JSON file, it should work. Make sure to put all your data between the `{}`, just like you would when making a dictionary.
+Make sure to put all your data between the `{}`, just like you would when making a dictionary.
```

Comment:
It should be clearer that an object is equivalent to a dictionary and an array is equivalent to a list.